components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,607 @@
+
+#### 3.3.3. Sea surface height gradient
+The sea surface height (SSH) gradient uses the standard gradient formulation from cell center to edge,
+
+$$
+-g\nabla(h_i+b_i) = -g\frac{1}{d_e} \sum_{i\in CE(e)} -n_{e,i}(h_i+b_i).
+$$
+
+#### 3.3.4. Del2 momentum dissipation
+The Del2, or Laplacian operator, viscous momentum dissipation maps edge-normal velocity back to the edge-normal component of the Laplacian. In TRiSK this is done with the "grad-div plus vorticity curl" formulation. (check grad omega term).
+
+$$
+\nu_2 \nabla^2 u_e = \nu_2 \left( \nabla \delta_i + {\bf k} \times \nabla \omega_v\right) = \frac{\nu_2}{d_e} \sum_{i\in CE(e)} -n_{e,i}\delta_i + \frac{\nu_2}{l_e} \sum_{v\in VE(e)} -n_{e,v}\omega_v
+$$
+
+#### 3.3.5. Del4 momentum dissipation
+The Del4, or biharmonic, momentum dissipation also maps edge-normal velocity back to the edge-normal component of the Laplacian. This is done with two applications of the Del2 operator above.
+
+$$
+- \nu_4 \nabla^4 u_e
+$$
+
+#### 3.3.6. Rayleigh Drag
+Rayleigh drag is simply linear drag, applied to all levels. It is typically only used during the spin-up process to damp large velocities during the initial adjustement. The Rayleigh coefficient $Ra$ is simply a scalar constant.
+
+$$
+- Ra \: u_e
+$$
+
+#### 3.3.7. Bottom drag
+Bottom drag is more relevant to layered models than to shallow water systems, but is included here for completeness. It is a quadratic drag applied to the edge velocity,
+
+$$
+- C_D \frac{u_e\left|u_e\right|}{[h_i]_e}.
+$$
+
+#### 3.3.8. Wind forcing
+Wind forcing has the same form as the bottom drag, but the forcing is the difference between the current velocity and the wind $u_W$, interpolated and projected to the edge normal direction.
+
+$$
+- C_W \frac{(u_W - u_e)\left|u_W - u_e\right|}{[h_i]_e}
+$$
+
+### 3.4 Thickness and Tracer Terms
+
+#### 3.4.1. Tracer advection
+
+There are many schemes available for tracer advection. Simple schemes include centered advection and upstream. MPAS-Ocean uses Flux Corrected Transport, which is fourth order under normal conditions, and reduces to third order to preserve monotinicity. 
+
+The tracer advection term is
+$$
+\nabla \cdot \left(u_e [h_i \phi_i]_e \right)
+$$
+
+and the stencil for divergence makes this
+$$
+\nabla \cdot \left( u_e [h_i \phi_i]_e \right)  =  \frac{1}{A_i} \sum_{e\in EC(i)} n_{e,i}u_e [h_i \phi_i]_e l_e
+$$
+The question is how to interpolate to the edge to obtain $[h_i \phi_i]_e$. Centered advection simply uses the midpoint rule. Upwind takes the value from the upstream cell. Higher order schemes cast a wider stencil to compute the value at this edge.
```

Comment:
We should add the higher order scheme here down the road.

Reply:
Agreed. I added a sentence that MPAS-Ocean used FCT.